.gitignore

```diff
@@ -1,2 +1,3 @@
 .DS_Store
 .vscode/
+src/src.ino

```

Comment:
i added that because sometimes i throw things into a sketch file there for quick testing, but i don't want it to be part of the library. since it's just a git thing it shouldn't affect how the library gets published.